changelogs/current.yaml

```diff
@@ -6,6 +6,11 @@ behavior_changes:
     OAuth filter now URL-encodes URL in query parameters. These query parameters are decoded, leaving intact character
     sequences that must remain encoded in URLs. This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.oauth_use_url_encoding`` to false.
+- area: oauth2
```

Comment:
nit: I think a note in the bug fixes section is enough.

Reply:
@yanavlasov, the entry has been removed from `behavior_changes`.